File: kitten-housing-care/src/main/java/org/jesperancinha/housing/converter/CareCenterConverter.java
package org.jesperancinha.housing.converter;

import org.jesperancinha.housing.data.CareCenterDto;
import org.jesperancinha.housing.model.CareCenter;

public class CareCenterConverter {
    public static CareCenterDto toDto(CareCenter careCenter) {
        return CareCenterDto.builder()
                .name(careCenter.getName())
                .address(careCenter.getAddress())
                .city(careCenter.getCity())
                .refNumber(careCenter.getRefNumber())
                .postCode(careCenter.getPostCode())
                .country(careCenter.getCountry())
                .build();
    }
}

File: kitten-house-care/src/main/kotlin/org/jesperancinha/housing/converter/CareCenterConverter.kt
package org.jesperancinha.housing.converter

import org.jesperancinha.housing.data.CareCenterDto
import org.jesperancinha.housing.model.CareCenter

fun CareCenter.toDto() = CareCenterDto(
    name = name,
    address = address,
    city = city,
    refNumber = refNumber,
    postCode = postCode,
    country = country
)